src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/filters/suppressionsinglefilter/Example1.java

```diff
@@ -1,20 +1,33 @@
 /*xml
 <module name="Checker">
   <module name="SuppressionSingleFilter">
-    <property name="checks" value="JavadocStyleCheck"/>
     <property name="files" value="AbstractComplexityCheck.java"/>
     <property name="lines" value="82,108-122"/>
```

Comment:
please restore `<property name="checks" value="JavadocStyleCheck"/>`
and just add this Check to Treewalker and make sure check find some violation. 

--------

files should Example1.java

We should not reference any other files except ExampleX.java

--------

> `<property name="lines" value="82,108-122"/>`

this values are weird.
Lets think on this how user read Example1.java, so lets  `<property name="lines" value="1,5-100"/>`
so it will mean suppress above class and somewhere in the middle of class.
In real execution line 1 will be on header, but it is ok, all will be suppressed by 5-100.